.gitlab-ci.yml

```diff
@@ -19,7 +19,7 @@ before_script:
 
 # ================== templates ================== #
 .base_template: &base_template
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:master
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:zach_update-jquery
```

Comment:
Is this correct?

Reply:
yeah, if something goes wrong we can just revert this PR, we don't also have to revert the `corp-ci` PR

Reply:
once this is merged in, we can change the image back to `master`